app/dashboard/template.hbs

```diff
@@ -152,7 +152,27 @@
                                             <div class='col-sm-12 text-center'>
                                                 <p>{{t 'dashboard.quicksearch.no_projects.line1'}}</p>
                                                 <p>{{t 'dashboard.quicksearch.no_projects.line2'}}</p>
-                                                <img src='/assets/images/dashboard/quicksearch-min.png' alt='{{t 'dashboard.quicksearch.no_projects.preview_alt'}}' class='img-responsive center-block'>
+
+                                                <div local-class='iframe-container'>
+                                                    <iframe
+                                                        local-class='osf-video'
```

Comment:
@brianpilati This will likely need a data-test selector. It usually is a four part nomenclature akin to 'data-test-descriptive-name'. If you have any questions, let me know but this usually helps Selenium tests run. For an element like this it is highly recommended.

Reply:
Actually, there is an example in the OsfLink below but still feel free to ask should you need.